src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -63,10 +65,11 @@ $VaultName= "MyKeyVault"
 $KeyVault = Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName
 $DiskEncryptionKeyVaultUrl = $KeyVault.VaultUri
 $KeyVaultResourceId = $KeyVault.ResourceId
-Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId
+$VolumeType = "All"
+Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId -VolumeType $VolumeType
 ```
 
-This example demonstrates enabling encryption without specifying AD credentials.   
+This example enables encryption on a Windows VM without specifying AD credentials.
```

Comment:
Not sure why this adds "on a Windows VM" since I think this would work for Linux too

Reply:
Removed "on Windows".